FtpClientApp/FtpClientApp/FileDownload.cs

```diff
@@ -1,10 +1,109 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace FtpClientApp
-{
-    class FileDownload
-    {
-    }
-}
+﻿using System;
+using System.IO;
+using System.Net;
+
+namespace FtpClientApp
+{
+    class FileDownload
+    {
+        public FileDownload(ServerConnectionInformation myConnection)
+        {   //for test
+            //myConnection.ServerName = "ftp://speedtest.tele2.net";
+            //myConnection.UserName = "anonymous";
+            //myConnection.PassWord = "anonymous";
+
+            string FileName = "1KB.zip";
+            FileName = getFileName(myConnection);
+
+            string DownloadDirectory = "c:/download";
+            DownloadDirectory = getDownLoadPath(FileName);
+
+            int bytesRead = 0;
+            byte[] buffer = new byte[2048];
+
+            FtpWebRequest myServerConnectionRequest = (FtpWebRequest)WebRequest.Create(myConnection.ServerName);
+            myServerConnectionRequest.Credentials = new NetworkCredential(myConnection.UserName, myConnection.PassWord);
+            myServerConnectionRequest.Proxy = null;
+            myServerConnectionRequest.UsePassive = true;
+            myServerConnectionRequest.UseBinary = true;
+            myServerConnectionRequest.KeepAlive = true;
+
+            myServerConnectionRequest.Method = WebRequestMethods.Ftp.DownloadFile;
+
+            Stream StreamReader = myServerConnectionRequest.GetResponse().GetResponseStream();

```

Comment:
A try/catch block would also be useful here (in case, for example, the server is down)